test/src/test/java/org/corfudb/infrastructure/SequencerBenchmark.java

```diff
@@ -0,0 +1,125 @@
+package org.corfudb.infrastructure;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.corfudb.protocols.wireprotocol.Token;
+import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.test.benchmark.AbstractCorfuBenchmark;
+import org.corfudb.test.benchmark.CorfuBenchmarkState;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+
+/**
+ * A set of benchmarks for the sequencer server.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+public class SequencerBenchmark extends AbstractCorfuBenchmark {
+
+    public static class SequencerBenchmarkState extends CorfuBenchmarkState {
+        /**
+         * A runtime to use during the iteration.
+         */
+        CorfuRuntime rt;
+
+        @Override
+        public void initializeIteration() {
+            super.initializeIteration();
+            rt = getNewRuntime();
+        }
+
+        UUID nextStream() {
+            return UUID.randomUUID();
+        }
+    }
+
+    /**
+     * Measure token acquisition (non-transactional) performance.
+     */
+    @Benchmark
+    @Warmup(iterations = 5)
+    @Measurement(iterations = 10)
+    @Threads(5)
+    public void getToken(SequencerBenchmarkState state, Blackhole bh) {
+        bh.consume(state.rt.getSequencerView().next(state.nextStream()));
+    }
+
+    /**
+     * Measure token query performance.
+     */
+    @Benchmark
+    @Warmup(iterations = 5)
+    @Measurement(iterations = 10)
+    @Threads(5)
+    public void queryToken(SequencerBenchmarkState state, Blackhole bh) {
+        bh.consume(state.rt.getSequencerView().next(state.nextStream()));
+    }
+
+    public static class TransactionalSequencerBenchmarkState extends CorfuBenchmarkState {
+        /**
+         * A runtime to use during the iteration.
+         */
+        CorfuRuntime rt;
+
+        @Param({"1", "10"})
+        int STREAMS;
```

Comment:
this variable is not a constant, please make it lower case

Reply:
also make it private

Reply:
I think we should enable Sonarcube. I feel we can better utilize our time.
I'll fix these.